packages/opencode/src/session/message-v2.ts

```
@@ -1456,6 +1466,10 @@ export function fromError(
           ).toObject()
         }
       } catch {}
+      // Last resort: a bare connection-dropped message (no errno code, not a
+      // structured stream error) is still a retryable transport disconnect.
+      const bareTransport = classifyBareTransportMessage(e)
```

Biome (2.5.0)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)